check.py

```diff
@@ -292,6 +292,9 @@ def run_gcc_tests():
     for t in sorted(os.listdir(shared.get_test_dir('example'))):
         output_file = 'example'
         cmd = ['-I' + os.path.join(shared.options.binaryen_root, 'src'), '-g', '-pthread', '-o', output_file]
+        # build with ASan if enabled on CI
+        if ('COMPILER_FLAGS' in os.environ and 'fsanitize=address' in str(os.environ['COMPILER_FLAGS'])):
+            cmd.insert(0, '-fsanitize=address')
```

Comment:
It should be fine to just append the flag to the end here.

Reply:
I once encountered an error complaining about sanitize not being added before the first library or something like that, so my reflex was to prepend it just to be sure :). Made a commit without these lines to check what'll happen, maybe it's not necessary at all.

Reply:
Appears that this indeed was just a random problem on my end and these lines are not necessary, perhaps induced by tinkering with the environment variables. Good catch 👍